components/google-cloud/google_cloud_components/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Components root."""
+
+from google.cloud import aiplatform
+from google_cloud_components.mb_sdk_component import utils
+
+DatasetCreateOp = utils.convert_method_to_component(aiplatform.Dataset.create)
```

Comment:
Flagging this to note we may further qualify these Ops with `AIPlatform` prefix in the future before release.

Reply:
good point, I'm switching from `mb_sdk_component` to  `aiplatfrom_component` for container and moving the module under `google_cloud_components.aiplatfrom`.